api/v1/kataconfig_webhook.go

```diff
@@ -38,8 +38,10 @@ var (
 func (r *KataConfig) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	clientInst = mgr.GetClient()
 
+	kataconfiglog.Info("SetupWebhookWithManager")
 	return ctrl.NewWebhookManagedBy(mgr).
 		For(r).
+		WithValidator(r).
```

Comment:
How did you find out ? Can you share a link in the commit message for the sake of knowledge sharing ?